examples/lastrev-next-starter-pnpm/packages/ui/src/Person/Person.fragment.graphql

```diff
@@ -23,6 +25,9 @@ fragment Person_BaseFragment on Person {
   hero {
     ...Hero_BaseFragment
   }
+  breadcrumbs {
+    ...Link_BaseFragment
+  }
```

Comment:
Do you want to keep breadcrumbs in the starter even if it's not used on Lively ?